src/import/ImportWavPack.cpp

```diff
@@ -251,12 +257,14 @@ ProgressResult WavPackImportFileHandle::Import(WaveTrackFactory *trackFactory, T
             itemLen = WavpackGetTagItemIndexed(mWavPackContext, i, NULL, 0);
             std::string item (itemLen + 1, '\0');
             WavpackGetTagItemIndexed(mWavPackContext, i, item.data(), itemLen + 1);
+            item.resize(itemLen);           // remove terminating NULL from std::string
```

Comment:
I'm not sure why do is this needed. This is safe here, though, so nothing critical

Reply:
I'm not a regular C++ programmer and just discovered that std::strings can include terminating NULLs. The issue this caused was that when the tag name had a NULL at the end it would no longer match the same string without a NULL. So, for example, this line did not work as intended:

https://github.com/audacity/audacity/blob/2c3c9d9919b778bb27cc94d99ece211d9686c12f/src/TagsEditor.cpp#L462